frontend/packages/insights-plugin/src/components/InsightsPopup/mappers.ts

```diff
@@ -48,8 +48,13 @@ export const mapMetrics = (response: PrometheusResponse): Metrics => {
       return null;
     }
     const metricName = response.data?.result?.[i]?.metric?.metric;
-    values[metricName] = parseInt(value, 10);
+    if (values[metricName] === -1 || values[metricName] === undefined) {
+      values[metricName] = parseInt(value, 10);
+    }
   }
 
   return values;
 };
+
+export const isInitState = (values) =>
+  !Object.values(values).reduce((cur, acc) => (cur === -1 ? acc && false : acc), true);
```

Comment:
`acc &&` is probably not needed there

Reply:
`&&` should be there

Reply:
I'll just quote here my previous answer to you:

> Nope, I'm saying that it's more clear version

I don't know where you saw that this works differently, but ok

Reply:
I'm confused here. Isn't `acc && false` always the same as `false`?

If I'm reading the logic right, can this be...
```suggestion
  Object.values(values).some((cur: number) => cur === -1);
```

Reply:
Fixed

Reply:
that one is much nicer!